source/_posts/2019-02-20-why-we-invest-in-having-our-teams-visit-clients.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Why we invest in having our teams visit clients
+comments: true
+author:
+  name: Sebastián Martínez
+  email: sebastian@wyeworks.com
+  twitter_handle: smartinez87
+  github_handle:  smartinez87
+  image:  /images/team/sebastian-martinez.jpg
+  description: VP of Engineering at WyeWorks. Love to travel the world and experience all the different foods and spices in it. Lifestyle medicine advocate.
+published: true
+---
+
+Remotism works great. The amount of different tools that came out in the last few years enabled people to work from any part of the world. You can have your code reviewed by your teammates from anywhere, jump on a call with people across different timezones, and so on. This allowed companies to hire talent without limiting the search to their zip code area, thus being able to get the best people regardless on where they live.
+
+That big leap in the working industry enabled us to work with great clients in the US, while keeping our offices in our home country. But in my opinion, remotism falls short compared to working side by side with other people in a few aspects that we need to be aware of, and find ways to compensate. When you're part of a team, in-person collaboration goes a long way. That is the reason why teams at WyeWorks are sent to work from our clients' offices every now and then for some time. We never stop to amaze ourselves about the amount of value those visits provide to both parties, and because of that they have become an instrumental aspect of our workflow.
+
+So, why do we invest in having our engineers fly several hours to visit the client?
+
+<!--more-->
+
+### We get precious context information
+
+When you're not there you miss a lot of the little details, and your head quickly tends to fill that void with assumptions. When people you're videoconferencing with usually arrive late to meetings, you automatically start assuming that maybe the meeting is not that important for them, or maybe they're just lazy. But when you're there with them, you get to see they might have to walk all the way across the office to get to the meeting room, and that the room is always being used by other people exceeding their time, and that setting things up for the call take a couple of extra minutes they already don't have.
```

Comment:
`When people you're videoconferencing with usually arrive late to meetings, you automatically start assuming that maybe the meeting is not that important for them`

This is kind of assuming that **all people always** assumes the wrong thing.. I think it's more like something that could happen (or something that usually happens?). I guess it could be fairest if rephrased using some conditional: e.g. "you may start assuming... "